README.md

```diff
@@ -1,3 +1,68 @@
+
+## MultiscaleRun
+
+MultiscaleRun is a Python package to run brain cells simulation at different scales.
+It orchestrates the coupling between several brain simulators like Neuron and
+STEPS but also solvers like AstroVascPy for the cerebral blood flow.
+The package also embeds a Julia solver to simulate the astrocytes activity.
+
+The Python package includes a program called `multiscale-run` that lets you run
+and analyze multiscale simulations from start to finish.
+
+## Testing for Development
+
+### Prerequisites
+- have an OBI spack installation working: https://github.com/openbraininstitute/spack
+- install julia with juliaup (not strictly necessary but it is harder to remove than to leave as is)
+
+### Setup
+
+You just need to run `setup.sh` at least once before running the simulation.
+
+The script does this:
+- set various env variables
+- create a `spackenv` folder with the necessary dependencies
+- create a `julia` project with the necessary dependencies in `juliaenv`
+- create a python virtual env in `venv` 
+- call `pip install -e .` for development
+- create the test folder `tiny_CI_test`
+- fill it with the necessary data
+
+```bash
+source setup.sh
+```
+
+If a folder is present (`spackenv`, `venv`, `juliaenv`) the script skips that installation step assuming that is already done. If any of the folders are missing, the script redoes the setup. 
+
+The environemnt is still set as it is needed. 
+
+You can always modify them and recall `setup.sh`. It will not override your changes. 
+
+### Test
+
+You just need to go to `tiny_CI_test` and run. The simulation is too slow with just one core. I suggest at least 8 cores. Do not go above 90 for now as this leaves some cores without neurons (edge case that I did not check). 
+
+```bash
+cd tiny_CI_test
+mpirun -np 8 multiscale-run compute
```

Comment:
Do we have some viewer that showcases the output so we see that it ran and did what was expected? Or is it too early and that will come later?

Reply:
we have post-processing but I did not actually check it. I am still fighting with julia